tests/system/test_instance_admin.py

```diff
@@ -105,14 +105,14 @@ def test_client_list_instances(admin_client, admin_instance_populated, not_in_em
     assert admin_instance_populated.name in found
 
 
-def test_instance_exists_hit(admin_instance_populated):
+def test_instance_exists_hit(admin_instance_populated, not_in_emulator):
```

Comment:
this is a little confusing - can we rename the variable to `in_emulator` and adjust accordingly?

Reply:
this has a side_effect of skipping if `not_in_emulator`.

Reply:
We could name this `skip_on_emulator` perhaps? Or something of the sort?